packages/web/src/hooks/useTabs/useTabs.tsx

```diff
@@ -21,6 +21,7 @@ import { throttle } from 'lodash'
 import { animated, useTransition, useSpring } from 'react-spring'
 import { useDrag } from 'react-use-gesture'
 
+import { ClientOnly } from 'components/client-only/ClientOnly'
```

Comment:
i wonder if we want to have `components/client-only` or `components/ssr` or something